src/edu/ucsb/cs56/networking/expression_evaluator/BasicServer.java

```diff
@@ -0,0 +1,55 @@
+package edu.ucsb.cs56.networking.expression_evaluator;
+import java.lang.*;
+import java.io.*;
+import java.net.*;
+
+public class BasicServer {
+
+	int portNum;
+	ServerSocket servSock;
+	Socket clientSock;
```

Comment:
You might want to make this an `ArrayList<Socket> clientSocks` in future commits in order to be able to communicate with multiple clients simultaneously